tests/alembic_migrations/versions/v_84b7fc2596d5_add_utility_views/conftest.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from sqlalchemy.orm import sessionmaker
+from ...alembicverify_util import prepare_schema_from_migrations
+from ....helpers import insert_crmp_data
+
+
+@pytest.fixture(scope='function')
+def prepared_schema_from_migrations_left(
+        uri_left, alembic_config_left, db_setup
+):
+    engine, script =  prepare_schema_from_migrations(
+        uri_left,
+        alembic_config_left,
+        db_setup=db_setup,
+        revision='84b7fc2596d5'
```

Comment:
I think it could be a good idea to make this a parameter for the fixture.  However, if the `conftest.py` files are not merged together it may not be necessary.

Reply:
I thought so too, but it did lead to some repetition, specifically that each test that uses the fixture will have to be parametrized correctly with the specific revision number. I thought I would just suck that up into a common fixture in the test subdirectory's conftest. I wanted to use a common fixture above it that is parametrized (using pytest fixture indirection, which is the only way I know how to do that), but I couldn't figure out how to supply a parameter to a fixture rather than to a test function. If you know how, I'd be delighted to learn.

Reply:
OTOH, inelegant as it is, it's only a few lines repeated.

P.S. Well spotted. The one thing I don't like about these tests!

Reply:
I think it would be nice, but if it isn't a simple fix then don't worry about it.  Like you said there isn't much repetition happening so it should be fine.